docs/icicle/rust-bindings.md

```diff
@@ -22,8 +22,72 @@ icicle-bn254 = { git = "https://github.com/ingonyama-zk/icicle.git", tag = "v1.0
 `icicle-bn254` being the curve you wish to use and `icicle-core` and `icicle-cuda-runtime` contain ICICLE utilities and CUDA wrappers.
 
 If you wish to point to a specific ICICLE branch add `branch = "<name_of_branch>"` or `tag = "<name_of_tag>"` to the ICICLE dependency. For a specific commit add `rev = "<commit_id>"`.
+ 
+Lets write a quick example to demonstrate how you would use the Rust ICICLE library. Create a file named `main.rs` in your project, with the following:
```

Comment:
Instead of having someone create the project themselves, would it be better to have them clone ICICLE and look at the rust ntt example that is there already?

We will need to keep the docs and the example code in sync as it is

Reply:
I already linked the examples, @LeonHibnik wanted this type of example